Wrapping/Generators/Python/itk/support/extras.py

```diff
@@ -801,6 +805,49 @@ def image_from_vtk_image(vtk_image: "vtk.vtkImageData") -> "itkt.ImageBase":
     return l_image
 
 
+def dict_from_image(image: "itkt.Image") -> Dict:
+    """Serialize a Python itk.Image object to a pickable Python dictionary."""
+    import itk
+
+    pixel_arr = itk.array_view_from_image(image)
+    dimension = image.GetImageDimension()
+    componentType, pixelType = image_to_type(image)
+    if 'int64' in componentType:
```

Comment:
We can remove this work around since this is used with WebAssembly and WebAssembly supports 64-bit integer arrays.